content/sensu-go/5.15/installation/auth.md

```diff
@@ -1288,7 +1288,7 @@ To use OIDC for authentication, register Sensu Go as an OIDC application. Use th
 #### Requirements
 
 - Access to the Okta Administrator Dashboard
-- Sensu Go 5.12.0 or later with a valid license
+- Sensu Go 5.12.0 or later with a valid commercial license
```

Comment:
A license is not required to access this feature.

Reply:
I agree that this needs some massaging to make sense in 5.15 and later. 

The original intent is to communicate that this is a commercial feature introduced in 5.12.0; in reality a license is required to use this feature on versions between 5.12.0 and 5.14.x.

Reply:
I revised to:
"Sensu Go 5.12.0 or later (plus a valid commercial license for Sensu Go versions 5.12.0 through 5.14.2)"

Wdyt?